registry/storage/driver/testsuites/testsuites.go

```diff
@@ -269,7 +269,7 @@ func (suite *DriverSuite) TestWriteReadStreamsNonUTF8(c *check.C) {
 	suite.writeReadCompareStreams(c, filename, contents)
 }
 
-// TestWriteReadLargeStreams tests that a 5GB file may be written to the storage
```

Comment:
This amount of data needs to be tested to handle assumptions made in certain storage drivers to handle large files.

This can be skipped with `-test.short`.